src/styles/main.scss

```diff
@@ -1,3 +1,46 @@
-body {
-  margin: 0;
+@import "reset";
+$size: 300px;
+
+.snake {
+  display: grid;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  grid-template-rows: repeat(6, $size);
+  text-align: center;
+  line-height: $size;
+
+  &__first {
+    background-color: #f00;
+  }
+
+  &__second {
+    background-color: #c00;
+  }
+
+  &__third {
+    background-color: #900;
+  }
+
+  &__fourth {
+    background-color: #600;
+  }
+
+  &__fifth {
+    background-color: #300;
+  }
+
+  &__sixth {
+    background-color: #000;
+  }
+
+  @media (min-width: 600px) {
+    grid-template-columns: repeat(2, auto);
+    grid-template-rows: repeat(3, $size);
+  }
+
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, auto);
+    grid-template-rows: repeat(2, $size);
```

Comment:
grid-template-rows is redundant here, rows are set by column number and size is set in .snake selector

Reply:
Thanks, your comments have been corrected